### DISCUSSION TITLE
Replacement for Google Cloud IoT Core

### DISCUSSION THREAD

**Discussion by mhfikri** [csobrinho/hydroponics#4] [General]
Hi @csobrinho , I'm pretty sure you already know that Google will retired Cloud IoT Core on 16th August this year, do you have plan to migrate this project to another platform?

**Comment by csobrinho** [comment]
Hi @mhfikri. Yes, unfortunately... I'm toying with two possibilities but haven't done any code yet:
1) make it generic as in any vanilla mqtt server. Potentially create the server side on a raspberry since it's mostly local. Have some sort of push to the raspberry from the app if needed. Soon I'll also start sending the values via UDP broadcast so I'll have 1 value/sec on the app. The local option actually saves a lot of money at the end of year. No cloud, no monthly payments, etc.

2) convert to Amazon AWS.

Biggest issue right now is finding a good replacement for the realtime database Firestore.

In sum, TBD!
